capture/capture.go

```diff
@@ -3,8 +3,6 @@ package capture
 import (
```

Comment:
the only changes I made to this file are formatting and linting related, namely to shut my IDE up lmao.